py2many/clike.py

```diff
@@ -208,6 +208,11 @@ def _typename_from_type_node(self, node) -> Union[List, str, None]:
             return get_id(node)
         elif isinstance(node, ast.Tuple):
             return [self._typename_from_type_node(e) for e in node.elts]
+        elif isinstance(node, ast.Attribute):
+            node_id = get_id(node)
+            if node_id.startswith("typing."):
```

Comment:
One could also write:

```
from typing import List as typ_list
```
A more general solution would involve some type of eval/__import__ of the statement above and then looking up typing.List in a dict, like what we're doing in class_for_typename/_func_for_lookup.

Could use remove_prefix post 3.9:

https://www.python.org/dev/peps/pep-0616/

Reply:
sure.  there are lots of problems, and this only solves one of them, and does so without the overhead of eval/import.  Created https://github.com/adsharma/py2many/issues/409 for a fallback to more difficult cases.